js/main.js

```diff
@@ -466,43 +490,40 @@ function constructSampleHTML(samples) {
 
                 <div class="flex 2xl:font-medium rounded-2xl bg-fuchsia-600/60 px-2 text-stone-50 w-auto">${sample.tasks}</div>
 
-              ${
-                sample.models.length > 0
-                  ? ` ${sample.models
-                      .map(
-                        (model) =>
-                          `<div class="flex 2xl:font-medium rounded-2xl bg-stone-600/80 px-2 text-stone-50 w-auto">${model}</div>`
-                      )
-                      .join("")}`
-                  : ``
-              }
+              ${sample.models.length > 0
+          ? ` ${sample.models
+            .map(
+              (model) =>
+                `<div class="flex 2xl:font-medium rounded-2xl bg-stone-600/80 px-2 text-stone-50 w-auto">${model}</div>`
+            )
+            .join("")}`
+          : ``
+        }
 
                   <div
                     class="flex 2xl:font-medium rounded-2xl bg-teal-600/80 px-2 text-stone-50">${sample.framework}
                   </div>
 
-                  ${
-                    sample.webApis.length > 0
-                      ? `
+                  ${sample.webApis.length > 0
+          ? `
                  ${sample.webApis
-                   .map(
-                     (api) =>
-                       `<div class="flex 2xl:font-medium rounded-2xl bg-indigo-600/80 px-2 text-stone-50">${api}</div>`
-                   )
-                   .join("")}`
-                      : ``
-                  }
-
-                  ${
-                    sample.devices.length > 0
-                      ? `${sample.devices
-                          .map(
-                            (device) =>
-                              `<div class="flex 2xl:font-medium rounded-2xl bg-sky-600/80 px-2 text-stone-50">${device}</div>`
-                          )
-                          .join("")}`
-                      : ``
-                  }
+            .map(
+              (api) =>
+                `<div class="flex 2xl:font-medium rounded-2xl bg-indigo-600/80 px-2 text-stone-50">${api}</div>`
+            )
+            .join("")}`
+          : ``
+        }
+
+                  ${sample.devices.length > 0
+          ? `${sample.devices
+            .map(
+              (device) =>
+                `<div class="flex 2xl:font-medium rounded-2xl bg-sky-600/80 px-2 text-stone-50">${device}</div>`
+            )
+            .join("")}`
+          : ``
+        }
```

Comment:
Could you please run `npm run format` in project root to update these formatting changes.

Reply:
Updated. However, I saw some format changes not caused by my PR, so I'm not sure if this is what you want.